Core/include/Acts/MagneticField/InterpolatedBFieldMap.hpp

```diff
@@ -200,10 +190,6 @@ class InterpolatedBFieldMap : public InterpolatedMagneticField {
     std::array<Vector3, nCorners> neighbors;
     const auto& cornerIndices = m_cfg.grid.closestPointsIndices(gridPosition);
 
-    if (!isInsideLocal(gridPosition)) {
-      return MagneticFieldError::OutOfBounds;
-    }
```

Comment:
I think this should be an exception or terminate the program.

Reply:
Right I just patched this up for now so it compiles. I wonder if the overflow bins can be used for this purpose?